### PR TITLE
Paginated list of albums

### LINKED ISSUE #2: Paginate list of albums

### PR DESCRIPTION
**Pull request by agentcap**
Fixed #2

### CODE REVIEW

File: src/app/services/spotify.service.ts [hunk 1]
```diff
@@ -27,8 +27,8 @@ export class SpotifyService{
 		return this._http.get(this.artistUrl).map(res => res.json());
 	}
 
-	getAlbums(name: string){
-		this.albumsUrl = 'http://ws.audioscrobbler.com/2.0/?method=artist.gettopalbums&artist='+name+'&api_key='+this.apiToken+'&format=json&limit=10';
+	getAlbums(name: string, limit: number){
+		this.albumsUrl = 'http://ws.audioscrobbler.com/2.0/?method=artist.gettopalbums&artist='+name+'&api_key=73a3a0eaa1cc2d3de22e5b6450ed6652&format=json&limit='+limit;
```
Comment (jajodiaraghav) [inline src/app/services/spotify.service.ts]: Why are you reverting this back?

**Comment by jajodiaraghav** [comment]
Can you please provide a screenshot/gif of the changes?

File: src/app/services/spotify.service.ts (hunk 1 above)
Comment (agentcap) [reply src/app/services/spotify.service.ts]: While resolving the conflicts I didn't check that it was replace by this.apiToken.I will update this.

**Comment by agentcap** [comment]
Pagination-Demo
A small demo of the pagination I implemented. Correct me if any changes are required.

File: src/app/services/spotify.service.ts (hunk 1 above)
Comment (agentcap) [reply src/app/services/spotify.service.ts]: And should I make the change and commit directly to the master branch?

Comment (jajodiaraghav) [reply src/app/services/spotify.service.ts]: Yes

**Comment by jajodiaraghav** [comment]
Great work Thanks @agentcap 
I would like to test this and see if any improvements can be made on this.
Cheers!

**Pull request merged**